frontend/app/routes/$lang+/_public+/status+/index.tsx

```diff
@@ -59,23 +70,27 @@ export async function action({ context: { session }, params, request }: ActionFu
   }
 
   const applicationStatusService = getApplicationStatusService();
+  const lookupService = getLookupService();
   const { sin, code } = parsedDataResult.data;
-  const status = await applicationStatusService.getStatusId(sin, code);
+  const statusId = await applicationStatusService.getStatusId(sin, code);
 
-  response.status = status ?? undefined;
+  const clientStatusList = await lookupService.getAllClientFriendlyStatuses();
+  const clientFriendlyStatus = clientStatusList.find((status) => status.id === statusId);
+
+  response.clientFriendlyStatus = clientFriendlyStatus ?? undefined;
+  response.alertType = clientFriendlyStatus?.id === CLIENT_STATUS_SUCCESS_ID ? 'success' : 'info';
   return json(response);
```

Comment:
Return `status` object that will include status information and the alert type. `as const` will help for typing the alert type correctly. Instead of type `string` it will be of literal type `"succes" | "info"`.

```
return json({
    status: {
      ...(clientFriendlyStatus ?? {}),
      alertType: clientFriendlyStatus?.id === CLIENT_STATUS_SUCCESS_ID ? 'success' : 'info',
    },
  } as const);
  ```